core/renderers/block_rendering_rewrite/measurables.js

```diff
@@ -587,3 +588,18 @@ Blockly.blockRendering.BottomRow.prototype.getNextConnection = function() {
   }
   return null;
 };
+
+Blockly.blockRendering.BottomRow.prototype.measure = function() {
+  for (var e = 0; e < this.elements.length; e++) {
+    var elem = this.elements[e];
+    this.width += elem.width;
+    if (!(elem.isSpacer())) {
+      if (elem.type == 'next connection') {
+        this.height = this.height + elem.height;
+        this.overhangY = elem.height;
```

Comment:
This assumes the next connection only extends down from the bottom. What if it extends up into the block as well?

Reply:
Nothing good.  Right now prev/next connections have to go down and output/input connections have to go left.  If we want to allow anything else I think I should make an issue for that so we can discuss.

Reply:
Yeah, please file that for discussion. I think it wouldn't be unreasonable to allow.